pkg/library/libraryapplyconfiguration/command.go

```diff
@@ -140,3 +146,16 @@ func (f *applyConfigurationFlags) ToOptions(ctx context.Context) (*applyConfigur
 		),
 		nil
 }
+
+func InitializeFeatureGates() error {
```

Comment:
I think an alternative to initializing the required FG for each binary is to add it directly to the manifestclient. The required FG could be initialized in the init() method. The benefit of this approach would be that the LabelSelectors would work automatically.

I'm fine with initialising per binary for now because I don't like when libraries have side effects.

Reply:
Moved to manifestclient